### PR TITLE
Fix Jobs icon

### PR DESCRIPTION
**Pull request by jfurrow**
I used the wrong icon before. Here's the new one:

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @pierlo-upitup

File: src/js/pages/JobsPage.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import mixin from 'reactjs-mixin';
 import React from 'react';
 import {RouteHandler} from 'react-router';
 
-import IconJobs from 'babel!svg-react!../../img/components/icons/pages.svg?name=IconJobs';
+import IconJobs from 'babel!svg-react!../../img/components/icons/pages-code.svg?name=IconJobs';
```
Comment (MatApple) [inline src/js/pages/JobsPage.js]: Are we going to load all SVGs like this? If so, maybe we should define the loaders in our webpack config so we don't have to prefix with `babel!svg-react!` here. What do you think?

Comment (jfurrow) [inline src/js/pages/JobsPage.js]: Oh good call! I didn't realize we could do that, but will get it in this PR.

**Comment by kennyt** [comment]
merge once mat's comment is done

File: src/js/pages/JobsPage.js (hunk 1 above)
Comment (jfurrow) [inline src/js/pages/JobsPage.js]: This is done now!

**Pull request merged**